### PR TITLE
Clarify system account edits which requires JetStream limits to be set to 0

### LINKED ISSUE #569: nsc edit is checking if JetStream is available
When you enter the following command:
`nsc edit account SYS -k OD.....2Y --tag A`
You will get the following error:
`Error: jetstream not available for system account`
NSC-1

### LINKED ISSUE #580: `nsc generate nkey` outputs good response to stderr rather than stdout
`nsc generate nkey` outputs a good response to stderr, rather than stdout, which is unexpected.      

```bash
root@instance:/# nsc generate nkey -a --store 2> e
root@instance:/# nsc generate nkey -a --store
AALS5NG2EJCJGCBTCRRH4PJO2R2UFHVAAIPLZEI5OPHHGW5T67DFNJRY
account key stored /root/.local/share/nats/nsc/keys/keys/A/AL/AALS5NG2EJCJGCBTCRRH4PJO2R2UFHVAAIPLZEI5OPHHGW5T67DFNJRY.nk
```

In particular, I would expect the following ansible to work.  But, to make it work i had to change stdout to stderr.

```yaml
    - name: Generate account signing key
      ansible.builtin.shell: >-
        nsc generate nkey -a --store
        {{ client_config }}
      register: result

    - name: Get the client account signing key 
      ansible.builtin.set_fact: client_account_signing_key="{{result.stdout_lines[0]}}"
```

### PR DESCRIPTION
**Pull request by aricart**
[FIX] clarified check added to edits of the system account which prevented editing due to JetStream default values in the flag - added a flag that sets the values for the jetstream setting (--js-disable) which sets the values to 0

Fixes #569

### CODE REVIEW

**Comment by bruth** [comment]
Is there any way to detect the account being edited is the system account for the selected/implicit operator?

**Comment by aricart** [comment]
> Is there any way to detect the account being edited is the system account for the selected/implicit operator?

Fixed - now if the --js-xxx flags are not specified, the edit to the account is performed. Otherwise the offending flag(s) are rejected.

**Review by bruth — APPROVED** [review]
LGTM!

File: cmd/editaccount.go [hunk 1]
```diff
@@ -236,6 +243,21 @@ func (p *EditAccountParams) getTierLimits(tier int) (*jwt.JetStreamLimits, error
 	return src, nil
 }
 
+func (p *EditAccountParams) doDisableJetStream(r *store.Report) error {
+	if !p.disableJetStream {
+		return nil
+	}
+	p.claim.Limits.JetStreamLimits = jwt.JetStreamLimits{}
+	r.AddOK("deleted global limit")
```
Comment (bruth) [inline cmd/editaccount.go:251]: "delete" may be misinterpreted, I suggest "cleared global limits" and "cleared tiered limit"

File: cmd/editaccount.go [hunk 2]
```diff
@@ -236,6 +243,21 @@ func (p *EditAccountParams) getTierLimits(tier int) (*jwt.JetStreamLimits, error
...
+	if p.claim.Limits.JetStreamTieredLimits != nil {
```
Comment (bruth) [inline cmd/editaccount.go:252]: The nil check is not needed for iteration.

**Review by bruth — APPROVED** [review]
Two minor suggestions, but otherwise LGTM

File: cmd/editaccount.go (hunk 2 above)
Comment (aricart) [reply cmd/editaccount.go]: fixed

File: cmd/editaccount.go (hunk 1 above)
Comment (aricart) [reply cmd/editaccount.go:251]: in other operations and tiers it is also labeled as "deleted" - so keeping it that way for now.

**Pull request merged**